### ISSUE TITLE
Some questions about the paper & experiments on large-scale pretrained models

### ISSUE THREAD

**Issue by jinhong-ni** [FutureXiang/ddae#2]
Dear authors,

Thanks for open-sourcing your amazing and inspiring work! I am really interested in applying generative diffusion models to discriminative tasks, and I would really appreciate if you could help me through a couple of concerns regarding your paper.

1. If I have understood everything correctly, the experimental results for say CIFAR-10 in your paper follow the steps: 1) train the generative diffusion models using the images from CIFAR-10; 2) find the optimal layer feature; 3) linear probe or fine-tune for classification using both the images and labels from CIFAR-10. I believe that since the model already sees all training images during pre-training, it already learns the image features within the dataset. Have you tried pre-training on one dataset, e.g. CIFAR-10, and transferring to another dataset e.g. TinyImageNet for downstream classification?
2. Following up on my previous question, have you tried or considered trying to replicate such experiments on large-scale pretrained diffusion models, e.g., Stable Diffusion? If this is not the case, what is the major gap in your opinion?

Thanks again for your great work, looking forward to your response!

**Comment by FutureXiang** [comment]
Hi, thanks for your interest in this work.
